examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -43,3 +46,19 @@
 
 # Now let's take a look at the spatial distributions of the psd.
 epochs.plot_psd_topomap(ch_type='grad', normalize=True)
+
+# Alternatively, you may also create PSDs from Epochs objects with psd_XXX
+f, ax = plt.subplots()
+psds, freqs = psd_multitaper(epochs[:5], low_bias=True, tmin=tmin, tmax=tmax,
```

Comment:
It was mostly just to speed things up. I didn't see why adding >5 epochs would improve the test and the multitaper psd takes longer than the welch psd. I can increase the epochs if need be, but it seemed like similar things were done elsewhere (e.g., only running it on a small number of channels)